plugins/com.google.cloud.tools.eclipse.preferences/schema/com.google.cloud.tools.eclipse.preferences.areas.exsd

```diff
@@ -0,0 +1,178 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="com.google.cloud.tools.eclipse.preferences" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appinfo>
+         <meta.schema plugin="com.google.cloud.tools.eclipse.preferences" id="com.google.cloud.tools.eclipse.preferences.areas" name="Preference Areas"/>
```

Comment:
Is this documented anywhere? I feel like a I should file a bug on this.

In general, I find the Eclipse editors incredibly annoying. I wish they'd show me the damn XML and get out of my way.